sdk/keyvault/keyvault-certificates/src/certificatesModels.ts

```diff
@@ -52,9 +64,63 @@ export interface CertificateWithPolicy extends Certificate {
 
 /**
  * @interface
- * An interface representing the attributes of a certificate
+ * An interface representing a certificate's policy
+ */
+export interface CertificatePolicy
+  extends SecretProperties,
+    X509CertificateProperties,
+    CertificateAttributes {
+  /**
+   * The certificate id.
+   * **NOTE: This property will not be serialized. It can only be populated by the server.**
+   */
+  readonly id?: string;
+  /**
+   * Actions that will be performed by Key Vault over the lifetime of a certificate.
+   */
+  lifetimeActions?: LifetimeAction[];
+  /**
+   * Indicates if the private key can be exported.
+   */
+  exportable?: boolean;
+  /**
+   * The type of key pair to be used for the certificate. Possible values include: 'EC', 'EC-HSM',
+   * 'RSA', 'RSA-HSM', 'oct'
+   */
+  keyType?: JsonWebKeyType;
+  /**
+   * The key size in bits. For example: 2048, 3072, or 4096 for RSA.
+   */
+  keySize?: number;
+  /**
+   * Indicates if the same key pair will be used on certificate renewal.
+   */
+  reuseKey?: boolean;
+  /**
+   * Elliptic curve name. For valid values, see JsonWebKeyCurveName. Possible values include:
+   * 'P-256', 'P-384', 'P-521', 'P-256K'
+   */
+  curveType?: JsonWebKeyCurveName;
```

Comment:
>curveType?: JsonWebKeyCurveName; [](start = 2, length = 32)

Should this be KeyCurveType since KeyType and KeySize are both prefixed with key?